Clean up Javadoc Warnings

When Pulsar builds in CI, I can see the below warnings printed out (https://github.com/apache/pulsar/runs/5622671591?check_suite_focus=true):

```
 28 warnings
Warning:  Javadoc Warnings
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/BrokerData.java:30: warning - invalid usage of tag <
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/BrokerData.java:30: warning - invalid usage of tag >
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java:536: warning - Tag @link: reference not found: ServiceConfiguration.loadBalancerBrokerMaxTopics
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java:417: warning - invalid usage of tag {@param namespaceName}
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java:536: warning - Tag @link: reference not found: ServiceConfiguration.loadBalancerBrokerMaxTopics
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java:42: warning - invalid usage of tag {@link LocalBrokerData#getMaxResourceUsageWithWeight)
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java:41: warning - Tag @link: reference not found: ServiceConfiguration#loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold()
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/WRRPlacementStrategy.java:50: warning - invalid usage of tag >
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java:292: warning - @return tag cannot be used in method with void return type.
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java:37: warning - invalid usage of tag <
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java:37: warning - invalid usage of tag <
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java:40: warning - invalid usage of tag >
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java:44: warning - invalid usage of tag >
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java:44: warning - invalid usage of tag >
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java:49: warning - invalid usage of tag >
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/TopicPoliciesService.java:86: warning - invalid usage of tag <
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/TopicPoliciesService.java:86: warning - invalid usage of tag >
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java:349: warning - invalid usage of tag <
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java:79: warning - invalid usage of tag &
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java:79: warning - invalid usage of tag &
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java:79: warning - invalid usage of tag &
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java:79: warning - invalid usage of tag &
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java:54: warning - invalid usage of tag {@link https://prometheus.io/docs/instrumenting/exposition_formats/
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/TransactionBuffer.java:39: warning - Tag @link: reference not found: #commitTxn(TxnID, long, long)
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java:536: warning - Tag @link: reference not found: ServiceConfiguration.loadBalancerBrokerMaxTopics
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java:79: warning - invalid usage of tag &
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java:79: warning - invalid usage of tag &
Warning:  /pulsar/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java:536: warning - Tag @link: reference not found: ServiceConfiguration.loadBalancerBrokerMaxTopics
```

If possible, it'd be great to resolve these warnings to clean up the build logs a little.

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/BrokerData.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/BrokerData.java
@@ -27,7 +27,7 @@ import org.apache.pulsar.policies.data.loadbalancer.LocalBrokerData;
  * Data class containing three components comprising all the data available for the leader broker about other brokers: -
  * The local broker data which is written to ZooKeeper by each individual broker (LocalBrokerData). - The time average
  * bundle data which is written to ZooKeeper by the leader broker (TimeAverageBrokerData). - The preallocated bundles
- * which are not written to ZooKeeper but are maintained by the leader broker (Map<String, BundleData>).
+ * which are not written to ZooKeeper but are maintained by the leader broker (Map&lt;String, BundleData&gt;).
  */
 @Data
 public class BrokerData {

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java
@@ -414,7 +414,6 @@ public class LoadManagerShared {
 
     /**
      * It returns map of broker and count of namespace that are belong to the same anti-affinity group as given.
-     * {@param namespaceName}
      *
      * @param pulsar
      * @param namespaceName
@@ -533,7 +532,7 @@ public class LoadManagerShared {
 
     /**
      * It filters out brokers which owns topic higher than configured threshold at
-     * {@link ServiceConfiguration.loadBalancerBrokerMaxTopics}. <br/>
+     * {@link: ServiceConfiguration#loadBalancerBrokerMaxTopics}. <br/>
      * if all the brokers own topic higher than threshold then it resets the list with original broker candidates
      *
      * @param brokerCandidateCache

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/LoadManagerShared.java
@@ -532,7 +532,7 @@ public class LoadManagerShared {
 
     /**
      * It filters out brokers which owns topic higher than configured threshold at
-     * {@link: ServiceConfiguration#loadBalancerBrokerMaxTopics}. <br/>
+     * ServiceConfiguration.loadBalancerBrokerMaxTopics. <br/>
      * if all the brokers own topic higher than threshold then it resets the list with original broker candidates
      *
      * @param brokerCandidateCache

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java
@@ -39,7 +39,7 @@ import org.slf4j.LoggerFactory;
  * Load shedding strategy that unloads any broker that exceeds the average resource utilization of all brokers by a
  * configured threshold. As a consequence, this strategy tends to distribute load among all brokers. It does this by
  * first computing the average resource usage per broker for the whole cluster. The resource usage for each broker is
- * calculated using the following method: {@link LocalBrokerData#getMaxResourceUsageWithWeight)}. The weights for each
+ * calculated using the following method: {@link: LocalBrokerData#getMaxResourceUsageWithWeight)}. The weights for each
  * resource are configurable. Historical observations are included in the running average based on the broker's
  * setting for loadBalancerHistoryResourcePercentage. Once the average resource usage is calculated, a broker's
  * current/historical usage is compared to the average broker usage. If a broker's usage is greater than the average

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ThresholdShedder.java
@@ -39,8 +39,9 @@ import org.slf4j.LoggerFactory;
  * Load shedding strategy that unloads any broker that exceeds the average resource utilization of all brokers by a
  * configured threshold. As a consequence, this strategy tends to distribute load among all brokers. It does this by
  * first computing the average resource usage per broker for the whole cluster. The resource usage for each broker is
- * calculated using the following method: {@link: LocalBrokerData#getMaxResourceUsageWithWeight)}. The weights for each
- * resource are configurable. Historical observations are included in the running average based on the broker's
+ * calculated using the following method:
+ * {@link LocalBrokerData#getMaxResourceUsageWithWeight(double, double, double, double, double)}. The weights
+ * for each resource are configurable. Historical observations are included in the running average based on the broker's
  * setting for loadBalancerHistoryResourcePercentage. Once the average resource usage is calculated, a broker's
  * current/historical usage is compared to the average broker usage. If a broker's usage is greater than the average
  * usage per broker plus the loadBalancerBrokerThresholdShedderPercentage, this load shedder proposes removing

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java
@@ -37,8 +37,8 @@ import org.apache.pulsar.policies.data.loadbalancer.LocalBrokerData;
 /**
  * This strategy tends to distribute load uniformly across all brokers. This strategy checks load difference between
  * broker with highest load and broker with lowest load. If the difference is higher than configured thresholds
- * {@link ServiceConfiguration#getLoadBalancerMsgRateDifferenceShedderThreshold()} or
- * {@link ServiceConfiguration#loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold()} then it finds out
+ * {@link: ServiceConfiguration#getLoadBalancerMsgRateDifferenceShedderThreshold()} or
+ * {@link: ServiceConfiguration#loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold()} then it finds out
  * bundles which can be unloaded to distribute traffic evenly across all brokers.
  *
  */

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/UniformLoadShedder.java
@@ -37,8 +37,8 @@ import org.apache.pulsar.policies.data.loadbalancer.LocalBrokerData;
 /**
  * This strategy tends to distribute load uniformly across all brokers. This strategy checks load difference between
  * broker with highest load and broker with lowest load. If the difference is higher than configured thresholds
- * {@link: ServiceConfiguration#getLoadBalancerMsgRateDifferenceShedderThreshold()} or
- * {@link: ServiceConfiguration#loadBalancerMsgThroughputMultiplierDifferenceShedderThreshold()} then it finds out
+ * {@link ServiceConfiguration#getLoadBalancerMsgRateDifferenceShedderThreshold()} or
+ * {@link ServiceConfiguration#getLoadBalancerMsgThroughputMultiplierDifferenceShedderThreshold()} then it finds out
  * bundles which can be unloaded to distribute traffic evenly across all brokers.
  *
  */

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/WRRPlacementStrategy.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/WRRPlacementStrategy.java
@@ -38,7 +38,7 @@ public class WRRPlacementStrategy implements PlacementStrategy {
     /**
      * Function : getByWeightedRoundRobin returns ResourceUnit selected by WRR algorithm
      *              based on available resource on RU.
-     * <code>
+     * &lt;code&gt;
      * ^
      * |
      * |
@@ -47,14 +47,14 @@ public class WRRPlacementStrategy implements PlacementStrategy {
      * |                |                        |                                |     |
      * |   Broker 2     |       Broker 3         |         Broker 1               |  B4 |
      * |                |                        |                                |     |
-     * +----------------+------------------------+--------------------------------+--------->
+     * +----------------+------------------------+--------------------------------+---------
      * 0                20                       50                               90    100
      *
      * This is weighted Round robin, we calculate weight based on availability of resources;
      * total availability is taken as a full range then each broker is given range based on
      *  its resource availability, if the number generated within total range happens to be in
      * broker's range, that broker is selected
-     * </code>
+     * &lt;/code&gt;
      */
     public ResourceUnit findBrokerForPlacement(Multimap<Long, ResourceUnit> finalCandidates) {
         if (finalCandidates.isEmpty()) {

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java
@@ -286,7 +286,6 @@ public class NamespaceService implements AutoCloseable {
     /**
      * Register all the bootstrap name spaces including the heartbeat namespace.
      *
-     * @return
      * @throws PulsarServerException
      */
     public void registerBootstrapNamespaces() throws PulsarServerException {

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/HashRangeAutoSplitStickyKeyConsumerSelector.java
@@ -34,19 +34,19 @@ import org.apache.pulsar.client.api.Range;
  * 2.The whole range of hash value could be covered by all the consumers.
  * 3.Once a consumer is removed, the left consumers could still serve the whole range.
  *
- * Initializing with a fixed hash range, by default 2 << 15.
+ * Initializing with a fixed hash range, by default 2 &gt;&gt; 15.
  * First consumer added, hash range looks like:
  *
- * 0 -> 65536(consumer-1)
+ * 0 -&lt; 65536(consumer-1)
  *
  * Second consumer added, will find a biggest range to split:
  *
- * 0 -> 32768(consumer-2) -> 65536(consumer-1)
+ * 0 -&lt; 32768(consumer-2) -&lt; 65536(consumer-1)
  *
  * While a consumer removed, The range for this consumer will be taken over
  * by other consumer, consumer-2 removed:
  *
- * 0 -> 65536(consumer-1)
+ * 0 -&lt; 65536(consumer-1)
  *
  * In this approach use skip list map to maintain the hash range and consumers.
  *

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/TopicPoliciesService.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/TopicPoliciesService.java
@@ -83,7 +83,7 @@ public interface TopicPoliciesService {
      * @param topicName topic name
      * @param backoff back off policy
      * @param isGlobal is global policies
-     * @return CompletableFuture-&lt;Optional-&lt;TopicPolicies-&gt;-&gt;
+     * @return CompletableFuture&lt;Optional&lt;TopicPolicies&gt;&gt;
      */
     default CompletableFuture<Optional<TopicPolicies>> getTopicPoliciesAsyncWithRetry(TopicName topicName,
               final Backoff backoff, ScheduledExecutorService scheduledExecutorService, boolean isGlobal) {

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/TopicPoliciesService.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/TopicPoliciesService.java
@@ -83,7 +83,7 @@ public interface TopicPoliciesService {
      * @param topicName topic name
      * @param backoff back off policy
      * @param isGlobal is global policies
-     * @return CompletableFuture<Optional<TopicPolicies>>
+     * @return CompletableFuture-&lt;Optional-&lt;TopicPolicies-&gt;-&gt;
      */
     default CompletableFuture<Optional<TopicPolicies>> getTopicPoliciesAsyncWithRetry(TopicName topicName,
               final Backoff backoff, ScheduledExecutorService scheduledExecutorService, boolean isGlobal) {

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java
@@ -346,7 +346,7 @@ public class DispatchRateLimiter {
     }
 
     /**
-     * Update dispatch rate by updating msg and byte rate-limiter. If dispatch-rate is configured -&lt; 0 then it closes
+     * Update dispatch rate by updating msg and byte rate-limiter. If dispatch-rate is configured &lt; 0 then it closes
      * the rate-limiter and disables appropriate rate-limiter.
      *
      * @param dispatchRate

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/DispatchRateLimiter.java
@@ -346,7 +346,7 @@ public class DispatchRateLimiter {
     }
 
     /**
-     * Update dispatch rate by updating msg and byte rate-limiter. If dispatch-rate is configured < 0 then it closes
+     * Update dispatch rate by updating msg and byte rate-limiter. If dispatch-rate is configured -&lt; 0 then it closes
      * the rate-limiter and disables appropriate rate-limiter.
      *
      * @param dispatchRate

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/service/streamingdispatch/StreamingEntryReader.java
@@ -76,7 +76,7 @@ public class StreamingEntryReader implements AsyncCallbacks.ReadEntryCallback, W
     /**
      * Read entries in streaming way, that said instead of reading with micro batch and send entries to consumer after
      * all entries in the batch are read from ledger, this method will fire numEntriesToRead requests to managedLedger
-     * and send entry to consumer whenever it is read && all entries before it have been sent to consumer.
+     * and send entry to consumer whenever it is read and all entries before it have been sent to consumer.
      * @param numEntriesToRead number of entry to read from ledger.
      * @param maxReadSizeByte maximum byte will be read from ledger.
      * @param ctx Context send along with read request.

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java
@@ -51,7 +51,7 @@ import org.apache.pulsar.common.util.SimpleTextOutputStream;
 /**
  * Generate metrics aggregated at the namespace level and optionally at a topic level and formats them out
  * in a text format suitable to be consumed by Prometheus.
- * Format specification can be found at @see <a href="https://prometheus.io/docs/instrumenting/exposition_formats/">Exposition Formats</a>
+ * Format specification can be found at <a href="https://prometheus.io/docs/instrumenting/exposition_formats/">Exposition Formats</a>
  */
 public class PrometheusMetricsGenerator {
 

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java
@@ -51,7 +51,7 @@ import org.apache.pulsar.common.util.SimpleTextOutputStream;
 /**
  * Generate metrics aggregated at the namespace level and optionally at a topic level and formats them out
  * in a text format suitable to be consumed by Prometheus.
- * Format specification can be found at {@link: https://prometheus.io/docs/instrumenting/exposition_formats/}
+ * Format specification can be found at @see <a href="https://prometheus.io/docs/instrumenting/exposition_formats/">Exposition Formats</a>
  */
 public class PrometheusMetricsGenerator {
 

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/PrometheusMetricsGenerator.java
@@ -51,7 +51,7 @@ import org.apache.pulsar.common.util.SimpleTextOutputStream;
 /**
  * Generate metrics aggregated at the namespace level and optionally at a topic level and formats them out
  * in a text format suitable to be consumed by Prometheus.
- * Format specification can be found at {@link https://prometheus.io/docs/instrumenting/exposition_formats/}
+ * Format specification can be found at {@link: https://prometheus.io/docs/instrumenting/exposition_formats/}
  */
 public class PrometheusMetricsGenerator {
 

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/TransactionBuffer.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/TransactionBuffer.java
@@ -36,8 +36,8 @@ import org.apache.pulsar.common.policies.data.TransactionInBufferStats;
  *
  * <p>When committing transaction starts, the broker will append a `COMMITTED`
  * marker to the data partition first to mark the transaction is committed.
- * The broker knows the data ledger of the commit marker and calls {@link: #commitTxn(TxnID, long, long)}
- * to commit and seal the buffer.
+ * The broker knows the data ledger of the commit marker and calls
+ * {@link TransactionBuffer#commitTxn(TxnID, long)} to commit and seal the buffer.
  *
  * <p>When the marker is appended to the data partition, all the entries are visible
  * to the consumers. So a transaction reader {@link TransactionBufferReader} will be

--- a/pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/TransactionBuffer.java
+++ b/pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/TransactionBuffer.java
@@ -36,7 +36,7 @@ import org.apache.pulsar.common.policies.data.TransactionInBufferStats;
  *
  * <p>When committing transaction starts, the broker will append a `COMMITTED`
  * marker to the data partition first to mark the transaction is committed.
- * The broker knows the data ledger of the commit marker and calls {@link #commitTxn(TxnID, long, long)}
+ * The broker knows the data ledger of the commit marker and calls {@link: #commitTxn(TxnID, long, long)}
  * to commit and seal the buffer.
  *
  * <p>When the marker is appended to the data partition, all the entries are visible

